src/pyfaf/retrace.py

```diff
@@ -1,5 +1,5 @@
 import re
-import concurrent.futures as futures
+import concurrent.futures as futures #pylint: disable=useless-import-alias
```

Comment:
Is pylint really complaining about this?

Reply:
It was (that's why I added the `disable` directive) though I can't remember when exactly. It could be either when running `make rpm` or when building the local docker image with `make build_local` in the `docker` directory.

Reply:
That's strange. I haven't been able to reproduce this, neither in faf nor on a reduced, standalone case.

Reply:
I think `from concurrent import futures` could be used as well, no?

Reply:
Yeah, fixed.